setup.php

```diff
@@ -92,6 +92,30 @@ public function enqueue_scripts($page_hook) {
     );
     wp_enqueue_script('yoast-cmb2-plugin-js');
   }
+
+
+  /**
+   * Test whether Yoast SEO is hidden either by choice of the admin or because
+   * the post type is not a public post type
+   *
+   *
+   * @param  string $post_type (optional) The post type to test, defaults to the current post post_type.
+   *
+   * @return  bool        Whether or not the yoast seo metabox should be hidden
+   */
+  function check_if_yoast_seo_is_hidden( $post_type = null ) {
```

Comment:
seeing as we are only calling this method inside this class, could we declare this method as `private` and also remove the extra white spaces?